[RESULT] [A.y] No two exceptions to ever overlap for this variable
[ERROR] [A.y] This variable might return an empty error:
 --> tests/test_proof/bad/no_vars-empty.catala_en
  | 
7 |   context y content integer
  |           ^
  + Test
The solver generated the following counterexample to explain the faulty behavior:
--> x : 1
[RESULT] [A.x] No two exceptions to ever overlap for this variable
[RESULT] [A.x] This variable never returns an empty error
